mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
```

Comment:
This function is already defined in the file context, redefining it will cause issues. Remove this duplicate function definition.